config/config.yaml

```
@@ -0,0 +1,6 @@
+defaults:
+  - dataset: nerf_synthetic
```

```suggestion
  - dataset: bunny_smoke
```
related to this https://github.com/ueda0319/neddf/pull/25/files/eea2d11e0b325558b55db8a73a6b89468a362bdd..6fa6b7c9c69a115d2444eef6a5b1441185f3acbb#r941915514